package.json

```diff
@@ -2,12 +2,16 @@
   "name": "pdf-generation-actions-web-component",
   "version": "1.11.0",
   "description": "components actions to print or render the report to a pdf",
-  "main": "dist/index.js",
+  "main": "index.js",
```

Comment:
We need to point to the build file not the source.

Reply:
Modified